kubernetes/cosmos/values.yaml

```diff
@@ -37,32 +38,28 @@ rclone:
     tag: 1.45
     pullPolicy: IfNotPresent
 
+  suspend: true
   schedule: "0 */12 * * *"
   successfulJobsHistory: 1
 
   remote:
-    accessKey: my-access-key
-    secretKey: my-secret-key
-    endpoint: http://cloudserver
-    region: pfsd-1
-    bucket: pfsd-1
+    credentials:
+      existingSercret: ""
+
+      accessKey: my-access-key
+      secretKey: my-secret-key
+    endpoint: http://zenko-cloudserver
+    region: mynfs
+    bucket: newbucket
 
   resources: {}
   nodeSelector: {}
   tolerations: []
   affinity: {}
 
-persistentVolume:
-  enabled: false
-  volumeConfig:
-    ## Example NFS config
-    # nfs:
-    #   server: 10.100.1.42
-    #   path: /data
-    #   readOnly: false
-    # mountOptions:
-    # - hard
-    # - nfsvers=4.1
-  accessModes:
-    - ReadWriteMany
-  size: 1Gi
+backend:
```

Comment:
I'm not a huge fan of obscuring the PV options like this because it directly blocks any other out of the box use-case (localFS, block, iSCSI, even SMB). The reason is because each type of PV will have completely different types of config values and can't account for every single one. For example, we have SMB listed however the parameters are in a completely different structure and will even vary based off the SMB FlexVolume plugin that is being used.

Also the way it is prior to these changes already allows for the future FTP use-case since PV is disabled by default and wouldn't need to add the `if not (eq .Values.backend.type "ftp"` all over.